### PR TITLE
improve documentation

### PR DESCRIPTION
**Pull request by scoopex**
Update CiaB documentation

### CODE REVIEW

File: docs/advanced-guides/cloud-in-a-box.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ sidebar_position: 10
 
 # Cloud in a Box - CiaB
 
-💡 Cloud in a Box (CiaB) is a minimalistic installation of OSISM with only services which are needed to
+💡 Cloud in a Box (CiaB) is a minimalistic installation of the latest OSISM with only services which are needed to
```
Comment (berendt) [inline docs/advanced-guides/cloud-in-a-box.md:8]: ... of the latest stable ...

File: docs/advanced-guides/cloud-in-a-box.md [hunk 2]
```diff
@@ -45,8 +45,14 @@ There are two types of Cloud in a Box.
 
 ## Installation
 
-1. Download one of the Cloud in a Box images of type sandbox
+:::warning
+The images currently download and install the 
+[latest release of the installation-scripts](https://github.com/osism/cloud-in-a-box),
+therefore it is mandatory to update the installation media when the base release changes.
```
Comment (berendt) [inline docs/advanced-guides/cloud-in-a-box.md:51]: This is not necessary. You only have to change the ISO image when we change the underlaying distribution (Ubuntu 22.04 -> Ubuntu 24.04). The ISO images does not have to be changed when there is a new OSISM release.

File: docs/advanced-guides/cloud-in-a-box.md [hunk 3]
```diff
@@ -45,8 +45,14 @@ There are two types of Cloud in a Box.
...
+The installation of historical releases is currently not supported. 
+:::
 
+1. Download one of the Cloud in a Box images of type sandbox</br>
```
Comment (berendt) [inline docs/advanced-guides/cloud-in-a-box.md:55]: Is there not better way than using HTML? I would prefer to have Markdown only.

File: docs/advanced-guides/cloud-in-a-box.md [hunk 4]
```diff
@@ -45,8 +45,14 @@ There are two types of Cloud in a Box.
 
 ## Installation
 
-1. Download one of the Cloud in a Box images of type sandbox
+:::warning
```
Comment (berendt) [inline docs/advanced-guides/cloud-in-a-box.md:48]: Not sure if it is required to place this information inside a box. Isn't it enough to just mention this in normal text?

**Review by berendt — CHANGES_REQUESTED** [review]

File: docs/advanced-guides/cloud-in-a-box.md (hunk 2 above)
Comment (scoopex) [reply docs/advanced-guides/cloud-in-a-box.md]: Removed the warning

File: docs/advanced-guides/cloud-in-a-box.md (hunk 3 above)
Comment (scoopex) [reply docs/advanced-guides/cloud-in-a-box.md]: Removed that

File: docs/advanced-guides/cloud-in-a-box.md (hunk 4 above)
Comment (scoopex) [reply docs/advanced-guides/cloud-in-a-box.md]: Removed the warning

File: docs/advanced-guides/cloud-in-a-box.md [hunk 5]
```diff
@@ -45,8 +45,12 @@ There are two types of Cloud in a Box.
 
 ## Installation
 
-1. Download one of the Cloud in a Box images of type sandbox
+The images currently download and install the 
+[latest release of the installation-scripts](https://github.com/osism/cloud-in-a-box),
+therefore it is mandatory to update the installation media at least when the underlying Ubuntu operating 
+system release changes.  The installation of historical releases is currently not supported.
```
Comment (berendt) [inline docs/advanced-guides/cloud-in-a-box.md:51]: I think we should mention here that this is not planed for the next at least 12 months. I think we will have Ubuntu 24.04 not before the Q3 of 2024.

**Review by berendt — CHANGES_REQUESTED** [review]

File: docs/advanced-guides/cloud-in-a-box.md [hunk 6]
```diff
@@ -45,8 +45,12 @@ There are two types of Cloud in a Box.
 
 ## Installation
 
-1. Download one of the Cloud in a Box images of type sandbox
+The images currently download and install the 
+[latest release of the installation-scripts](https://github.com/osism/cloud-in-a-box),
```
[the remark this answers is not in the document]
Comment (scoopex) [reply docs/advanced-guides/cloud-in-a-box.md:49]: Yes.

File: docs/advanced-guides/cloud-in-a-box.md (hunk 5 above)
Comment (scoopex) [reply docs/advanced-guides/cloud-in-a-box.md:51]: Yes

**Pull request merged**